tests/test_ticker_gains.py

```diff
@@ -0,0 +1,34 @@
+from capgains.ticker_gains import TickerGains
+import pytest
+
+
+def test_ticker_gains_negative_balance(acb_transactions):
+    """ If the first transaction added is a sell, it is illegal since
+    this causes a negative balance, which is impossible"""
+    for transaction in acb_transactions:
+        if transaction.action == 'SELL':

```

Comment:
What's the point of this `if` statement here?

Reply:
Only run the test on SELL transactions, since they are supposed to fail